.gitignore

```diff
@@ -0,0 +1,4 @@
+/bin
+vendor
+composer.lock
+composer.phar
```

Comment:
why composer.phar ? this file should not be in the folder so not in the gitignore